src/test/regress/expected/multi_cluster_management.out

```diff
@@ -354,7 +352,7 @@ BEGIN;
 SELECT master_add_node('localhost', :worker_2_port);
            master_add_node           
 -------------------------------------
- (14,14,localhost,57638,default,f,t)
+ (13,13,localhost,57638,default,f,t)
```

Comment:
to make the tests more robust to change, it would be nice to hide the groupid where possible. e.g. `SELECT 1 FROM master_add_node(...)`